lib/rubocop/cop/style/raise_args.rb

```diff
@@ -21,6 +39,33 @@ def on_send(node)
 
         private
 
+        def autocorrect(node)
+          _scope, method, *args = *node
+
+          new_exception = if style == :compact
+                            correction_exploded_to_compact(args)
+                          else
+                            correction_compact_to_exploded(args)
+                          end
+          replacement = "#{method} #{new_exception}"
+
+          ->(corrector) { corrector.replace(node.source_range, replacement) }
+        end
+
+        def correction_compact_to_exploded(node)
```

Comment:
Actually, I started with that, but decided to remove the imperative verb, since the method returns the corrected source, and doesn't actually do the correction itself (after refactoring to reduce duplication.)

Other options could be `#corrected_compact_to_exploded` or `#replacement_compact_to_exploded`?